specification/compute/resource-manager/readme.az.md

```diff
@@ -21,13 +67,12 @@ cli:
             op: "*"
           hidden: false
         - where:
-            group: "*"
-            op: "*"
-            param: vmName
-          alias: name
```

Comment:
this setting for vmName=>name is necessary for depth coverage

Reply:
Added now.